tests/test_arrays.py

```diff
@@ -19,7 +19,11 @@
 from conftest import LightningDevice as ld
 from conftest import device_name, lightning_ops
 
-if device_name == "lightning_gpu" or not ld._CPP_BINARY_AVAILABLE:
+if (
+    device_name == "lightning_gpu"
+    or device_name == "lightning.tensor"
```

Comment:
Note that device names should have a dot separator, so I'm unsure whether this test is supposed to run for L-GPU.
```suggestion
    device_name in ("lightning.gpu", "lightning.tensor")
```